### ISSUE TITLE
Lost all the project

### ISSUE THREAD

**Issue by mariemgeo** [tilemill-project/tilemill#2468]
Hi everyone!

I'm working with tilemill (TileMill-v0.10.1) on windows 8 and osm datas with postgis database.

First, even if I work with a very little extent (few french departments) and layer after layer, the data loading and rendering is very lengthy.
Frequently, TileMill loads during five or ten minutes with no result. So I close it. Then, I wait the end of all TileMill process before I restart it. It produces again this morning but this time, I had this error message:

[tilemill] [tilemill] skipped loading project: Error: Could not open project.mml file for "name_of_project". Error was:
[tilemill]
[tilemill] "Unexpected end of input"
[tilemill]
[tilemill] (in C:/path of tilemill project)
[tilemill] at Function.<anonymous> (C:/Program Files/Tilemill-v0.10.1/tilemill/models/Project.server.bones:185:29)
[tilemill] at next (C:/Program Files/Tilemill-v0.10.1/tilemill/node_modules/step/lib/step.js:51:23)
[tilemill] at C:/Program Files
[tilemill] at C:/Program Files/Tilemill-v0.10.1/tilemill/models/Project.server.bones:172:31
[tilemill] at C:/Program Files/Tilemill-v0.10.1/tilemill/lib/fsutil.js:21:13
[tilemill] at fs.readFile (fs.js:176:14)
[tilemill] at fs.close (C:/Program Files/Tilemill-v0.10.1/tilemill/node_modules/npm/node_modules/graceful-fs/graceful-fs.js:90:5)
[tilemill] at fs.close  (C:/Program Files/Tilemill-v0.10.1/tilemill/node_modules/glob/node_modules/graceful-fs/graceful-fs.js:90:5)
[tilemill] at Object.oncomplete (fs.js:297:15)
Please report bla bla bla

In fact during the last crash down of TileMill, I lost all the contents of the project.mml and other .mss files. I didnt lost the files and the project directory but the content.

Is someone had this problem soon?
Did I have a chance to get back the files content?

Thanks for help me.
Marie

**Comment by springmeyer** [comment]
Marie. thanks for letting us know about this. it has happened (the lost content) to one other user but we've been unable to replicate the problem and so unable to fix it. it is very rare. infortunel there is no way to recover the contents of the project.mml unless you have a backup. 

as far as the slowness the problem is that TileMill is waiting on Postgres queries. we have designed the new Mapbox Studio (tilemill successor) to solve this performance problem. are you interested in trying Mapbox Studio?

**Comment by Komzpa** [comment]
sorry, not actionable. would you please try again with new tilemill from master and open another issue if problem persists?
